frontend/templates/product-list/sections/HeroSection.tsx

```diff
@@ -72,7 +72,6 @@ const HeroTitle = chakra(
             as="h1"
             className={className}
             size="xl"
-            fontFamily="Archivo Black"
```

Comment:
I think this branch may be out of date- there are more than 3 references to `fontFamily="Archivo Black"` on main.

Reply:
Yep, Probably a couple more reference got created in the meantime
I'll sort this out tomorrow morning